### PR TITLE
Publish CodeChica++ lesson 0x04

### PR DESCRIPTION
**Pull request by xlgmokha**
# Why is this needed?

I need to publish this to the website so that my class know what they are
supposed to learn today and so that I know what I'm supposed to teach them.

## What does this do?

It adds a link to the new lesson page and it updates the agenda for that lesson.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ layout: default
 <a href="https://www.eventbrite.com/e/code-chica-advanced-coding-program-cohort-2-saturdays-tickets-215146768777" class="button primary">Register Now</a>
 -->
 
+gab c was here!
```
Comment (xlgmokha) [inline index.md:9]: Who is `gabi c` and why were they here?

**Comment by AlisaSalikhov** [comment]
Who is who

File: index.md (hunk 1 above)
Comment (AquaWaterfall23) [reply index.md:9]: :)

Comment (GabiCar2) [inline index.md:9]: this is true because gabi c was here! she told me herself

**Review by GabiCar2 — APPROVED** [review]
i approve

**Pull request merged**